lib/errors.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+class TldrError extends Error {
+  constructor(message) {
+    super(message);
+    this.name = 'TldrError';
+  }
+
+  static isTldrError(err) {
```

Comment:
Can we make this a global function rather than a static method?

Reply:
Sure but why? It'd just pollute the global namespace without any good reason?

Reply:
Ok.